frontend/next.config.js

```diff
@@ -8,7 +8,9 @@ const isProdBuild = process.env.NODE_ENV === nodeEnv.PRODUCTION;
 
 const PLAUSIBLE_URL = "https://plausible.io";
 
-const SCRIPT_SRC = ["'self'", PLAUSIBLE_URL];
+const TWITTER_URL = "https://cdn.syndication.twimg.com platform.twitter.com";
```

Comment:
just in case, I added `https://` to the second domain too 😆 

```suggestion
const TWITTER_URL = "https://cdn.syndication.twimg.com https://platform.twitter.com";
```